hail/src/main/scala/is/hail/lir/InitializeLocals.scala

```diff
@@ -13,7 +13,7 @@ object InitializeLocals {
     var i = entryUsedIn.nextSetBit(0)
     while (i >= 0) {
       val l = locals(i)
-      if (!l.isInstanceOf[Parameter]) {
+      if (!l.isInstanceOf[Parameter] && l.name != "spills") {
```

Comment:
🤔

Reply:
Oops, that wasn't supposed to make it in.  Deleted!